creator-node/src/middlewares/premiumContent/premiumContentMiddleware.ts

```diff
@@ -25,16 +27,20 @@ import config from '../../config'
  * If all these verifications are successful, then this middleware will proceed with the request as normal.
  */
 export const premiumContentMiddleware = async (
-  req: Request,
+  req: CustomRequest,
   res: Response,
   next: NextFunction
 ) => {
   const cid = req.params?.CID
+  const prometheusResult = { result: '', mode: 'default' }
+
   if (!cid) {
-    return sendResponse(
+    prometheusResult.result = 'abort_no_cid'
```

Comment:
all good for this, but made a card since this should not be coupled with premium content
https://linear.app/audius/issue/CON-465/move-cid-validation-out-of-premiumcontentmiddleware-into-its-own

Reply:
great, this can be a followup task to check general cid validity mw